Classes/Controller/PostController.php

```diff
@@ -149,7 +148,7 @@ protected function initializeView(ViewInterface $view): void
      */
     public function listRecentPostsAction(): void
     {
-        $maximumItems = (int)ArrayUtility::getValueByPath($this->settings, 'lists.posts.maximumDisplayedItems', '.') ?: 0;
+        $maximumItems = (int) (isset($this->settings['lists']['posts']['maximumDisplayedItems']) ? $this->settings['lists']['posts']['maximumDisplayedItems'] : 0);
```

Comment:
I'd go with `$maximumItems = (int)($this->settings['lists']['posts']['maximumDisplayedItems'] ?? 0);`